external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -43,8 +43,16 @@
           AfterTargets="RestorePackages"
           Condition="'$(TargetGroup)'=='uap'" >
 
+    <PropertyGroup>
+      <UAPToolsPackageVersion>1.0.1</UAPToolsPackageVersion>
+      <UAPToolsPackageName>Microsoft.DotNet.UAP.TestTools</UAPToolsPackageName>
+
+      <UAPToolsFolder>$(PackagesDir)$(UAPToolsPackageName)\$(UAPToolsPackageVersion)\Tools</UAPToolsFolder>
```

Comment:
I would still allow for someone to set this locally so I would add a condition for if this is empty set it.